integration-cli/registry_mock.go

```diff
@@ -36,7 +36,7 @@ func newTestRegistry(c *check.C) (*testRegistry, error) {
 		for re, function := range testReg.handlers {
 			matched, err = regexp.MatchString(re, url)
 			if err != nil {
-				c.Fatalf("Error with handler regexp")
+				c.Fatal("Error with handler regexp")
```

Comment:
Came to my mind the `return` in the next line isn't necessary after a `Fatal` (I'll leave the removal for a beginner issue cause I'm afk now if you all agree :))